### ISSUE TITLE
Arc Warden "Tempest" Flux bug 7.33

### ISSUE THREAD

**Issue by dezilxtoo4** [ValveSoftware/Dota2-Gameplay#8335]
### Ability name

Flux

### Description

Description in update:
Flux - Tempest version provides higher damage (15/30/45/60 -> 20/45/70/95) but lower movement speed penalty to the enemy (14/21/28/35% -> 6/10/14/18%)

But in fact we have 0% move speed penalty on any level of ability, also any level of "tempest double". Also it work same, if Arc Warden play on dire side, when he have "Tempest" version of ability's as main.

https://user-images.githubusercontent.com/95438587/233777622-4d25d99f-f435-4cde-860b-892af4aba41a.mp4

### Example Match ID (and possibly Timestamp)

-

**Comment by TheZett** [comment]
Tempest/Dire Warden Flux doesnt slow and the Magnetic Field talent doesnt upgrade the Tempest/Dire Warden Magnetic Field bonus damage at all.

**Comment by dezilxtoo4** [comment]
> Tempest/Dire Warden Flux doesnt slow and the Magnetic Field talent doesnt upgrade the Tempest/Dire Warden Magnetic Field bonus damage at all.

yep, but tempest flux should slow move speed according to the description (lower movement speed penalty to the enemy (14/21/28/35% -> 6/10/14/18%) 6-18%, not 0%

and strange mechanic of 3nd ability "spark wraith" with aghanim scepter on "Dire" Arc Warden, then sparks activates and fly faster with lower damage, 2nd "tempest" spark work like normal spark (2s delay before activate, and default fly speed) BUT damage lower like it is "tempest" spark :)

**Comment by Pflanzmann** [comment]
To add to this: Magnetic Field of the tempest gives enemies the damage if you have a shard

**Comment by Pflanzmann** [comment]
Also: The tempest bubble does not give evasion

**Comment by AlanKinder** [comment]
To add to this: Magnetic Field (with Aghanim Shard) from the Tempest gives Lina (hero) Fiery Soul stacks while inside it.
